packages/mui-material/src/SpeedDialAction/SpeedDialAction.js

```diff
@@ -169,13 +169,15 @@ const SpeedDialAction = React.forwardRef(function SpeedDialAction(inProps, ref)
   const fab = (
     <SpeedDialActionFab
       size="small"
-      className={clsx(classes.fab, className)}
       tabIndex={-1}
       role="menuitem"
       ownerState={ownerState}
+      {...other}
       {...FabProps}
```

Comment:
`FabProps` and `other` basically do the same thing now, but since `FabProps` is more specific I decided to give it preference over `other`. Same with the `style` attribute: `Fab.style` is applied last to win.